mod_bot/conf.py

```diff
@@ -0,0 +1,81 @@
+import os
+import json
+import pprint
+
```

Comment:
Maybe config could be our first object? It would stored the internal config instead of passing it in each methods that needs to read the conf

Reply:
i'm not a fan of "global variables", I think it's better to use DI (dependency injection) and therefore pass the configuration object in every method who needs it